src/OrchardCore.Modules/OrchardCore.Localization/AdminMenu.cs

```diff
@@ -25,23 +25,25 @@ public AdminMenu(IStringLocalizer<AdminMenu> localizer)
         ///<inheritdocs />
         public Task BuildNavigationAsync(string name, NavigationBuilder builder)
         {
-            if (String.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
+            if (!String.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
```

Comment:
I'm not sure doing this will improve execution time since it needs to evaluate that "if" still. But for code consistency, I guess it is not bad either.

Reply:
I don’t think it’ll effect execution time. The statement is still have to be evaluated. It just consistent and reduce indentation